molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/file/FileTestConfig.java

```diff
@@ -0,0 +1,37 @@
+package org.molgenis.integrationtest.file;
+
+import org.molgenis.file.FileStore;
+import org.molgenis.file.model.FileMetaFactory;
+import org.molgenis.file.model.FileMetaMetaData;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import java.io.File;
+
+@Configuration
+@Import({ FileMetaFactory.class, FileMetaMetaData.class })
+public class FileTestConfig
+{
+	@Bean
+	public FileStore fileStore()
+	{
+		// get molgenis home directory
+		String currentDir = System.getProperty("user.dir");
```

Comment:
user.dir is not the molgenis home directory. Why not create a random folder in the OS temp dir?